chart/githubconnector/templates/deployment.yaml

```diff
@@ -0,0 +1,29 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ .Values.name }}
+  namespace: {{ .Values.namespace }}
+  labels:
+    app: {{ .Values.name }}
+    chart: {{ include "github-connector-chart.chart" . }}
+spec:
+  selector:
+    matchLabels:
+      app: {{ .Values.name }}
```

Comment:
Don't use values as selector, because it may be shared between releases. Helm suggests using fullname: https://helm.sh/docs/chart_best_practices/#podtemplates-should-declare-selectors